docs/post_process.md

```diff
@@ -0,0 +1,43 @@
+# Post processing
+
+After a release is generated, all commits that contain an issue reference can
+be processed to update an external source with details of the version release.
+This allows issues in jira, notion etc to be updated with the version it was
+released as a part of.
```

Comment:
worth adding a link to https://nrwldev.github.io/changelog-gen/configuration/#post_process ?
(if/when fragments work)